packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -176,30 +179,30 @@ These APIs are compliant with the OpenID Connect and OAuth 2.0 spec with some Ok
 | insufficient_scope   | The access token provided doesn't contain the necessary scopes to access the resource.           |
 | login_required       | The request specified that no prompt should be shown but the user is currently not authenticated. |
 
-#### Request Examples
-This request initiates the authorization code flow, as signaled by `response_type=code`. The request returns an authorization code that you can use as the `code` parameter in a token request.
+#### Request examples
+This request initiates the authorization code flow as signaled by `response_type=code`. The request returns an authorization code that you can use as the `code` parameter in a token request.
 
 ```
 https://{yourOktaDomain}/oauth2/default/v1/authorize?client_id=0oabucvy
 c38HLL1ef0h7&response_type=code&scope=openid&redirect_uri=http%3A%2F%2Flocal
 host%3A8080&state=state-296bc9a0-a2a2-4a57-be1a-d0e2fd9bb601&nonce=g5ly497e8ps'
 ```
 
-This request does the same thing, but uses the `request` parameter to deliver a signed (HS256) JWT that contains all the query parameters:
+This request does the same thing, but uses the `request` parameter to deliver a signed (HS256) JWT that contains all of the query parameters:
 
 ```
 https://{yourOktaDomain}/oauth2/default/v1/authorize?
   request=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJPa3RhIiwiaWF0IjoxNTEyNTE2MjIxLCJleHAiOjE1NDQwNTIyMjEsImF1ZCI6Ind3dy5leGFtcGxlLmNvbSIsInN1YiI6InNqYWNrc29uQGV4YW1wbGUuY29tIiwiRW1haWwiOiJzamFja3NvbkBleGFtcGxlLmNvbSIsInJlc3BvbnNlX3R5cGUiOiJjb2RlIiwicmVzcG9uc2VfbW9kZSI6ImZvcm1fcG9zdCIsInJlZGlyZWN0X3VyaSI6Im15UmVkaXJlY3RVUkkuY29tIiwic3RhdGUiOiJteVN0YXRlIiwibm9uY2UiOiJteU5vbmNlIiwic2NvcGUiOiJvcGVuaWQgb2ZmbGluZV9hY2Nlc3MifQ.TjPy2_nUerULClavBNHcpnO_Pd1DxNEjQCeSW45ALJg"
 ```
 
-This request initiates the implicit flow, which gets an ID token and access token from the authorization server without the code exchange step. We use the same request as the first example, but with `response_type=id_token token`:
+This request initiates the implicit flow, which gets an ID token and access token from the Authorization Sserver without the code exchange step. We use the same request as the first example, but with `response_type=id_token token`:
```

Comment:
A bit too ssibilant!

Reply:
This is why Susan shouldn't do these things at the end of the day! Fixing....